x-pack/plugins/observability_solution/observability_onboarding/common/aws_firehose.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const FIREHOSE_CLOUDFORMATION_STACK_NAME = 'Elastic-CloudwatchLogsAndMetricsToFirehose';
+export const FIREHOSE_STREAM_NAME = 'Elastic-AllCloudwatchLogsAndMetrics';
+
+export const FIREHOSE_CLOUDFORMATION_TEMPLATE_URL =
+  'https://mykolaharmash-cf-template.s3.us-west-2.amazonaws.com/default_start.yml';
```

Comment:
Replace before merging. [The template PR](https://github.com/elastic/obs-cloudformation/pull/5).